### PR TITLE
Preferences edits

### PR DESCRIPTION
**Pull request by parenthetic**
Mostly improved description of how safety multipliers work, with a table of examples.

### CODE REVIEW

File: docs/docs/While You Wait For Gear/preferences-and-safety-settings.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Your preferences are found in the directory `myopenaps/preferences.json`.  To ed
 
 To check your edits when you're done, use `cd ~/myopenaps && cat preferences.json`
 
-## Commonly-adjusted preferences:
+## Available preferences to set:
```
Comment (scottleibrand) [inline docs/docs/While You Wait For Gear/preferences-and-safety-settings.md]: These are just the most commonly adjusted preferences: there are others not (yet?) documented here.

Comment (parenthetic) [reply docs/docs/While You Wait For Gear/preferences-and-safety-settings.md]: I interpreted "Commonly-adjusted" as "Frequently-adjusted," which seemed at odds with the safety multiplier settings I was making edits to. 

I'm not aware of what other preferences there are to set; but if there are others, the first sentence of this doc page seems incorrect.

Not sure how to correct...

**Comment by parenthetic** [comment]
Ugh. So embarrassing. I hadn't seen those preferences at the bottom of the document (tl;dr). The note at the top will be super helpful.

**Comment by danamlewis** [comment]
No worries @parenthetic! Thanks so much for this. Please keep them coming! :)

**Comment by parenthetic** [comment]
De nada. Thank you!!!

**Pull request merged**